src/lv_interop.h

```diff
@@ -96,6 +96,22 @@ namespace grpc_labview
     using LV1DArrayPtr = LV1DArray*;
     using LV1DArrayHandle = LV1DArray**;
 
+    struct LV2DArray {
+        int32_t dimensionSizes[2]; /* number of T elements for each dimension */
+        int8_t rawBytes[1]; /* (firstDimensionSize * secondDimensionSize * sizeof(T)) bytes */
+
+        // The start of 2D array data is always aligned and does not require padding.
```

Comment:
This is true for 1/2/4/8 but not 16+. Is it possible to static_assert that alignof(T) <= alignof(LV2DArray)?

Reply:
We discussed this offline, I think asserting `sizeof(T) <= 8` is fine

Reply:
Added assert for size of element being <= 8